### PR TITLE
Improve the coverage for the SectionChanger Component

### PR DESCRIPTION
**Pull request by HunseopJeong**
[Issue] N/A
[Problem] SectionChanger coverage is low
[Solution] Add TC cases to improve the coverage

Signed-off-by: Hunseop Jeong <hs85.jeong@samsung.com>

### CODE REVIEW

**Comment by HunseopJeong** [comment]
Before:
image

After:
image

File: tests/js/core/widget/core/SectionChanger/SectionChanger.js [hunk 1]
```diff
@@ -25,8 +25,92 @@
...
+			var element = document.getElementById("emptysectionchanger"),
+				sectionChanger = new SectionChanger();
+
+			sectionChanger.element = element;
+			sectionChanger._build(element);
+			sectionChanger._init(element);
+
+			expect(2);
+
+			assert.equal(sectionChanger.activeIndex, 0, "activeIndex is horizontal");
```
Comment (TomaszLukawskiSam) [inline tests/js/core/widget/core/SectionChanger/SectionChanger.js]: The `_init` method is much more complicated. In my opinion this test actually not testing whole `_init' method. You should remove this test.

Comment (HunseopJeong) [reply tests/js/core/widget/core/SectionChanger/SectionChanger.js]: I know what you're talking about. I will remove it.

**Review by TomaszLukawskiSam — APPROVED** [review]
Looks good to me.

**Review by singa2000 — APPROVED** [review]
It looks good.

**Pull request merged**